jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitProjects.kt

```diff
@@ -41,12 +66,25 @@ class SamHelloWorldGradle : SamProjectTemplate() {
     override fun dependencyManager(): String? = "gradle"
 }
 
-class SamDynamoDBCookieCutter : SamProjectTemplate() {
+abstract class SamPythonProjectTemplate : SamProjectTemplate() {
+    override fun supportedRuntimes() = setOf(Runtime.PYTHON2_7, Runtime.PYTHON3_6, Runtime.PYTHON3_7)
+
+    override fun postCreationAction(runtime: Runtime, contentRoot: VirtualFile, rootModel: ModifiableRootModel) {
+        super.postCreationAction(runtime, contentRoot, rootModel)
+        SamCommon.setSourceRoots(contentRoot, rootModel.project, rootModel)
+    }
+}
+
+class SamHelloWorldPython : SamPythonProjectTemplate() {
```

Comment:
this is definitely better than it was - ultimately it'll become untenable as we get more templates. I think it's fine for now - but at some point we're prolly going to have to have this more config driven.

Reply:
Agree